daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -1232,6 +1233,15 @@ private[lf] final class Compiler(
       }
     }
 
+  private[this] def compileInterfacePrecond(
+      iface: Identifier,
+      param: ExprVarName,
+      expr: Expr,
+  ) =
+    topLevelFunction1(t.InterfacePrecondDefRef(iface)) { (argPos, env) =>
+      compile(env.bindExprVar(param, argPos), expr)
+    }
```

Comment:
```suggestion
    topLevelFunction1(t.InterfacePrecondDefRef(iface))((argPos, env) =>
      compile(env.bindExprVar(param, argPos), expr)
    )
```

Reply:
probably my fault if most of the usage of `topLevelFunction1` used curly brackets.